src/Api.php

```diff
@@ -232,4 +242,20 @@ public function json(ResponseInterface $response) {
     return json_decode($response->getBody());
   }
 
+  public function sendWebHook($message)
```

Comment:
Maybe look at building a Drupal service for Slack or using something like https://www.drupal.org/slack which probably does this for you. This feels pretty unrelated to the rest of this API file. I feel It would be good to make notification more pluggable so users could use other services like HipChat and receive the same benefit. See also: https://www.drupal.org/docs/8/api/logging-api/overview